jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -359,7 +413,8 @@ public boolean equals(Object other) {
       return false;
     }
     ContainerConfiguration otherContainerConfiguration = (ContainerConfiguration) other;
-    return creationTime.equals(otherContainerConfiguration.creationTime)
+    return Objects.equals(platforms, otherContainerConfiguration.platforms)
```

Comment:
nit: since `platforms` is not `@Nullable`, no need to use `Objects.equals()`:
```suggestion
    return platforms.equals(otherContainerConfiguration.platforms)
```